types/event_bus.go

```diff
@@ -229,6 +233,10 @@ func (b *EventBus) PublishEventValidatorSetUpdates(data EventDataValidatorSetUpd
 	return b.Publish(EventValidatorSetUpdatesValue, data)
 }
 
+func (b *EventBus) PublishEventFastSyncUpdate(data EventDataValidatorSetUpdates) error {
+	return b.Publish(EventFastSyncStatusValue, data)
+}
+
```

Comment:
Why do we have this function and:
```
func (b *EventBus) PublishEventFastSyncStatus(data EventDataFastSyncStatus) error {
	return b.Publish(EventFastSyncStatusValue, data)
}
```
Are they not duplicates?

Reply:
ah, I will remove this, thanks for the catching